models/action_test.go

```diff
@@ -431,7 +400,7 @@ func TestUpdateIssuesCommit_AnotherRepoNoPermission(t *testing.T) {
 	AssertNotExistsBean(t, commentBean)
 	AssertNotExistsBean(t, issueBean, "is_closed=1")
 	assert.NoError(t, UpdateIssuesCommit(user, repo, pushCommits, repo.DefaultBranch))
-	AssertExistsAndLoadBean(t, commentBean)
+	AssertNotExistsBean(t, commentBean)
```

Comment:
Why this changed?

Reply:
Because this is `TestUpdateIssuesCommit_AnotherRepoNoPermission()`; the user has in fact no rights to create that comment (it was created before, but it was wrong IMHO).

Reply:
Because this is TestUpdateIssuesCommit_AnotherRepoNoPermission(); the user has in fact no rights to create that comment (it was created before, but it was wrong IMHO).